tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest2/small_3x3rod_HFP.inp

```diff
@@ -0,0 +1,443 @@
+***********************************************************************************************
```

Comment:
are we OK with putting this input file in the repo?
are we OK from an export control point of view?